helm-charts/namespace-cleaner/templates/_helpers.tpl

```diff
@@ -0,0 +1,32 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
do we have to mousage every line? Can we not do 
{{ /* Expand the name of hte chart.*/
define "namespace-cleaner.name
default .....
}}